sci-geosciences/qgis/qgis-9999.ebuild

```diff
@@ -17,48 +18,47 @@ fi
 inherit cmake desktop python-single-r1 qmake-utils xdg
 
 DESCRIPTION="User friendly Geographic Information System"
-HOMEPAGE="https://www.qgis.org/en/site/"
+HOMEPAGE="https://www.qgis.org/"
 
 LICENSE="GPL-2+ GPL-3+"
 SLOT="0"
-IUSE="3d examples georeferencer grass hdf5 mapserver netcdf opencl oracle polar postgres python qml"
+IUSE="3d examples georeferencer grass hdf5 mapserver netcdf opencl oracle polar postgres python qml webkit"
```

Comment:
We're avoiding adding new qtwebkit dependencies given it seems inactive upstream, so I'll drop this part.